### ISSUE TITLE
No GpC methylation calling?

### ISSUE THREAD

**Issue by yangxin-9** [hasindu2008/f5c#141]
Dear developer,
I was wondering that f5c does have a model for GpC methylation calling?

Best,
yang

**Comment by hasindu2008** [comment]
Is it for r9.4.1 chemistry or latest r10.4.1 you are interested in?

**Comment by yangxin-9** [comment]
Thanks for the reply.
    Nanopolish have a model for GpC methylation calling, but it does not support R10.4 flowcells, So I wonder if the f5c have the GpC model to supports R10.4.1.

**Comment by hasindu2008** [comment]
f5c currently does not have a model for GpC for R10.4.1. The CpG model was trained by us using the method described in  https://hasindu2008.github.io/f5c/docs/r10train.  The control dataset we sequenced for this training was https://zymoresearch.eu/products/human-methylated-non-methylated-wga-dna-set-dna-w-primers, and it only has CG sites methylated. If there is a suitable control sample like this for GC that you are aware of, we could try to train a model, as the method would be exactly the same.

**Comment by yangxin-9** [comment]
Thank you for the answer.
 I don't have a suitable control sample yet.

best wishes,
yangxin

**Comment by ShannonTown** [comment]
Hi, I was not sure from the posts above whether f5c has a model for r9.4.1 GpC calling? Or for both CpG and GpC? Thank you!

**Comment by hasindu2008** [comment]
@ShannonTown 
f5c does not have an in-built model for r9.4.1 GpC calling. But the model in nanopolish https://github.com/jts/nanopolish/blob/master/etc/r9-models/r9.4_450bps.gpc.6mer.template.model is possible to be adapted with a bit of work. What are you after?

**Comment by ShannonTown** [comment]
Thank you for your prompt reply! Inspired by your response, I also found
the models here:
https://github.com/jts/nanopolish/tree/nanonome/etc/r9-models
We're trying to do some in vitro exp to map nucleosome positions. But we
realized that the flow cells we use are r10.. I just tried using a dorado
basecall r10 model with `--modified-bases 5mC_5hmC` and it seemed to work.
Then I guess I need to separate the CpG and GpC meth somehow.
